build.gradle.kts

```diff
@@ -139,6 +139,28 @@ tasks.register("buildManageUsersApiModel", GenerateTask::class) {
 
 val generatedProjectDirs = listOf("locationsinsideprisonapi", "activitiesappointmentsapi", "prisonapi", "manageusersapi")
 
+tasks.register("integrationTest", Test::class) {
```

Comment:
This does mean you will need to run these separately locally going forwards.

Reply:
We could do a combined task to `./gradlew validate` (which runs targets `test, integrationTest`)? As a later step.